### PR TITLE
Remove unnecessary cast() in graph_data.py

### PR DESCRIPTION
**Pull request by junholee6a**
Issue: https://github.com/capitalone/DataProfiler/issues/717

Removing this `cast()` does not cause a mypy error.

By the time this line is reached, the type of `self.input_file_path` is `builtins.str`. This is because the `assert` statement at line 196, earlier in the function, removes the `None` type from `self.input_file_path`. Therefore this `cast()` doesn't do anything.

### CODE REVIEW

**Comment by taylorfturner** [comment]
hold off merging until #803 is merged. Thx

**Comment by taylorfturner** [comment]
once #800 is merge, we will move to this one to update branch, review, and merge @junholee6a

**Review by JGSweets — APPROVED** [review]

**Comment by taylorfturner** [comment]
okay @junholee6a read for an update branch on this one

**Pull request merged**
